lab_1_keywords_tfidf/main.py

```diff
@@ -66,8 +91,20 @@ def get_top_n(frequencies: dict[str, Union[int, float]], top: int) -> Optional[l
 
     In case of corrupt input arguments, None is returned
     """
-    pass
-
+    if not frequencies or not top:
```

Comment:
what if top is negative?

Reply:
should i add a condition that top mustn't be less than zero?

Reply:
yep, or even that top must be positive

Reply:
you know, to combine two different check into one 
increases readability

Reply:
done